views/home.tmpl

```diff
@@ -10,186 +10,222 @@
+                                    <span class="pl-1 unit lh15rem">Ph/s</span>
                                 </div>
-                            </td>
-                        </tr>
-                        <tr class="align-baseline">
-                            <td class="text-right pr-2 lh1rem">PROJECT</td>
-                            <td>
-                                <div class="mono lh1rem p03rem0 fs14-decimal fs18">
-                                    <span data-target="homepage.bsubsidyDev" class="d-inline-block">{{template "decimalParts" (amountAsDecimalParts .NBlockSubsidy.Dev true)}}</span
-                                    ><span class="pl-1 unit lh15rem">DCR</span>
+                                <div class="fs12 text-black-50 lh1rem text-black-50">
+                                    <span data-target="homepage.hashrateDelta">{{template "fmtPercentage" .HashRateChange}}</span> in past 24h
                                 </div>
-                            </td>
-                        </tr>
-                        <tr class="align-bottom">
-                            <td class="text-right pr-2 lh1rem pt-1 pb-1">PROOF OF WORK DIFFICULTY</td>
-                            <td>
-                                <div class="mono lh1rem p03rem0 fs24 fs14-decimal">
-                                    <span data-target="homepage.difficulty" >{{template "decimalParts" (float64AsDecimalParts .Difficulty 8 true)}}</span>
+                            </div>
+                            <div class="col-6 col-sm-4 col-md-6 col-lg-4 mb-3 mb-sm-2 mb-md-3 mb-lg-3">
+                                <div class="fs13 text-secondary">PoW Reward</div>
+                                <div class="mono lh1rem p03rem0 fs14-decimal fs24 p03rem0 d-flex align-items-baseline">
+                                    <span data-target="homepage.bsubsidyPow">{{template "decimalParts" (float64AsDecimalParts (toFloat64Amount .NBlockSubsidy.PoW) 8 true 2)}}</span>
+                                    <span class="pl-1 unit lh15rem">DCR</span>
                                 </div>
-                            </td>
-                        </tr>
-                        {{if .DevFund}}
-                        <tr class="align-baseline">
-                            <td class="text-right pr-2 lh1rem"><a href="/address/{{.DevAddress}}?txntype=merged_debit">TREASURY</a></td>
-                            <td>
-                                <div class="mono lh1rem p03rem0 fs14-decimal fs18">
-                                    <span data-target="homepage.devFund" class="d-inline-block">{{template "decimalParts" (amountAsDecimalParts .DevFund true)}}</span
-                                    ><span class="pl-1 unit lh15rem">DCR</span>
+                            </div>
+                            <div class="col-6 col-sm-5 col-md-6 col-lg-4 mb-3 mb-sm-2 mb-md-3 mb-lg-3">
```

Comment:
This one is `col-sm-5`, but all the others are `col-sm-4`. Is that intentional?

Reply:
Yes, since there's free space to the right of it at that breakpoint, giving it a bit more width prevents the label from wrapping, which in turn uses less vertical space in the layout.